x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/dimension_popover.tsx

```diff
@@ -3,6 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import './dimension_popover.scss';
```

Comment:
Why renaming and reimporting it here?
The new class could be used in the component as is

Reply:
I have no idea how I did put it here 😅  thanks for cautiousness!

Reply:
```suggestion
```

Reply:
This is the new standard for adding SASS stylesheets. Since SASS imports in JS can now compile automatically with all the EUI globals. This helps with not having to apply styles if the component doesn't exist on the page. 
https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#sass-files

Reply:
It was removed being imported in the `_index.scss` file.

Reply:
@mbondyra  You'll need to add this import back in. It was removed from the SASS blob `_index.scss` file. 

https://github.com/elastic/kibana/pull/75635/files?file-filters%5B%5D=.md&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-05cc06b27f9d592dd8ca7be25251b69bL2

I did this intentionally. I'm starting to convert SASS imports to the JS way as I touch them.

Reply:
I went ahead and reverted that last commit the deleted this line. So that CI's build would be correct.